web/client/codechecker_client/cmd/cmd.py

```diff
@@ -1430,5 +1458,28 @@ def add_arguments_to_parser(parser):
     login.set_defaults(func=cmd_line_client.handle_login)
     __add_common_arguments(login, needs_product_url=False)
 
+    export = subcommands.add_parser(
+        'export',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description="Export data (comments, review statuses) from a running "
+        "CodeChecker server into a json format",
```

Comment:
Do not we need to indent this text here?
```suggestion
        description="Export data (comments, review statuses) from a running "
                    "CodeChecker server into a json format",
```